sal_interfaces/Guider/Guider_Events.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="http://lsst-sal.tuc.noao.edu/schema/SALEventSet.xsl"?>
+<SALEventSet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALEventSet.xsd">
+  <Enumeration>GuiderStatus_Stopped,GuiderStatus_Starting,GuiderStatus_Guiding,GuiderStatus_Error</Enumeration>
+  <SALEvent>
+    <Subsystem>Guider</Subsystem>
+    <EFDB_Topic>Guider_logevent_guidingStatus</EFDB_Topic>
+    <Description>Event published when a guiding operatation starts, stops or resumes. It contains the parameters used to configure the guider (see startGuiding command) and the current status.</Description>
+    <item>
+      <EFDB_Name>roiX</EFDB_Name>
+      <Description>Vector with 8 elements specifying the center of the guider ROI window in the x-axis (in pixels) for each guider sensor. If an element of the array is zero, that guider will be ignored.</Description>
+      <IDL_Type>long</IDL_Type>
+      <Units>unitless</Units>
+      <Count>8</Count>
```

Comment:
See above. Unit shall be pix, either change type to float or specify ROI as a rectangle (topleft/upperright pixels)